src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TargetElementAttribute.cs

```diff
@@ -53,5 +53,29 @@ public TargetElementAttribute(string tag)
         /// <c>*</c> at the end of an attribute name acts as a prefix match.
         /// </remarks>
         public string Attributes { get; set; }
+
+        /// <summary>
+        /// The expected tag structure.
+        /// </summary>
+        /// <remarks>
+        /// If <see cref="TagStructure.Unspecified"/> (the default) and no other tag helpers applying to the same
+        /// element specify their <see cref="TagStructure"/> the <see cref="TagStructure.NormalOrSelfClosing"/>
+        /// behavior is used:
+        /// <code>
+        /// &lt;my-tag-helper&gt;&lt;/my-tag-helper&gt;
+        /// &lt;!-- OR --&gt;
+        /// &lt;my-tag-helper /&gt;
+        /// </code>
+        /// Otherwise, if another tag helper applying to the same element does specify their behavior, that behavior
+        /// is used.
+        ///
```

Comment:
Here and in `TagHelperDescriptor`: Blank lines don't change the final help. Instead use `<para>` elements to wrap the separate bits.